docs/README.md

```diff
@@ -0,0 +1,21 @@
+
+## Current Diagrams
+- [binding-controller](https://github.com/redhat-appstudio/integration-service/blob/main/docs/binding-controller.md)
+- [scenario-controller](https://github.com/redhat-appstudio/integration-service/blob/main/docs/scenario-controller.md)
+- [snapshot -controller](https://github.com/redhat-appstudio/integration-service/blob/main/docs/snapshot-controller.md)
+
+## Creating or editing Mermaid diagrams
+
+Mermaid is a JS based diagramming tool that renders markdown style syntax to create/modify diagrams. Mermaid has [native support in Github](https://github.com/github/roadmap/issues/372)
+
+## Editing Diagrams
+- Diagrams are stored in repository's `/docs` folder in the repository. As changes are made to the behaviour of controllers these changes should also be updated in the diagram. 
+- The updated mermaid diagram can be submitted as part of a seperate PR or part of the PR that changes the behaviour of the controller.. 
```

Comment:
nit:
```suggestion
- The updated mermaid diagram can be submitted as part of a separate PR or part of the PR that changes the behaviour of the controller.
```

Reply:
done

Reply:
I can still see the typo: seperate -> separate

Reply:
done

Reply:
thanks!